earth_enterprise/SConstruct

```diff
@@ -24,6 +26,27 @@ from khEnvironment import khEnvironment
 
 EnsureSConsVersion(0, 96, 92)
 
+# Check for the presents of the passed argument on
+# the command line.  If exists return a formated
+# string that can be passed to gradle else return
+# an empty string
+def check_for_argument( argument ) :
+  result = ""
+  if int( ARGUMENTS.get( argument, '0' ) ) > 0 :
+    result = "-P" + argument
+  return result
+
+# Check for command line arguments passed that must be
+# passed on to gradle
+def gradle_options() :
+  options = ""
+  options = check_for_argument( 'debug' )
+  options = check_for_argument( 'release' )
```

Comment:
On the other hand, `src/SConstruct` would make a debug build when `release=0`, but your code would currently not pass a `release` property to Gradle in that case.